pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -63,6 +64,7 @@ var (
 		noderestriction.PluginName,
 		sccadmission.PluginName,
 		storageclassdefaultadmission.PluginName,
+		expandpvcadmission.PluginName,
```

Comment:
You should be before the limit ranger check since limit ranges can be applied to PVC.

Reply:
> You should be before the limit ranger check since limit ranges can be applied to PVC.

Oh, its validating only.  Doesn't matter then.

Reply:
Yeah, this plugin just validates and does not mutate any of resources it validates.